Source/WebCore/style/AnchorPositionEvaluator.cpp

```diff
@@ -543,4 +565,47 @@ void AnchorPositionEvaluator::findAnchorsForAnchorPositionedElement(Ref<const El
     anchorPositionedState->stage = AnchorPositionResolutionStage::FoundAnchors;
 }
 
+void AnchorPositionEvaluator::updateSnapshottedScrollOffsets(Document& document)
+{
+    // https://drafts.csswg.org/css-anchor-position-1/#scroll
+
+    auto& states = document.styleScope().anchorPositionedStates();
+    for (auto elementAndState : states) {
+        auto& anchorPositionedElement = elementAndState.key;
```

Comment:
Can we use Ref here instead?
https://ews-build.s3-us-west-2.amazonaws.com/macOS-Safer-CPP-Checks-EWS/e09966ca-846/scan-build-output/StaticAnalyzerRegressions/WebCore/StaticAnalyzerReports/report-AnchorPositionEvaluator.cpp-updateSnapshottedScrollOffsets-5-408e86.html#EndPath